data/phi0588/abo001/__cts__.xml

```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0588.abo001" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0588">
             <ti:title xml:lang="lat">Miltiades</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi0588.abo001.perseus-lat2" workUrn="urn:cts:latinLit:phi0588.abo001">
-              <ti:label xml:lang="eng">Miltiades, Cornelii Nepotis Vitae</ti:label>
-              <ti:description xml:lang="eng">Nepos, Cornelius, creator; Fleckeisen, Alfred, 1820-1899, editor; Halm, Karl, 1809-1882, original editor</ti:description>
+              <ti:label xml:lang="lat">Miltiades</ti:label>
+              <ti:description xml:lang="mul">Nepos, Cornelius. Cornelii Nepotis Vitae. Fleckeisen, Alfred; Halm, Karl, editors.
```

Comment:
We generally have common name here, so "Cornelius Nepos." versus "Nepos, Cornelius."

Reply:
Ok, I will do that!